android/modules/ui/src/java/ti/modules/titanium/ui/TableViewProxy.java

```diff
@@ -189,8 +189,10 @@ private void appendRowInternal(Object rows, KrollDict animation, boolean interna
 				row.getProperties().optString(TiC.PROPERTY_FOOTER_TITLE,
 					row.getProperties().getString(TiC.PROPERTY_FOOTER)));
 
+			if (row.getParent() != null) {
```

Comment:
Please remove this conditional code as it would cause more issues because it's changing the parent-section of a row from one to another without properly removing the row from its previous parent-section first.

Reply:
But then we have the same code as before and it won't behave the same way it did in 12.5.1 where you were able to set content without running `update`.  Can you create a PR with a proper solution for both cases